tasks.py

```diff
@@ -215,47 +215,85 @@ def load_osm_streets(ctx, files=[]):
-            --connection-string {ctx.es} \
-            {additional_params} \
-            --dataset {ctx.dataset}".format(
-                ctx=ctx, additional_params=additional_params
-            ),
-        )
+    if not ctx.get("new-importer", False):
+        addr_config = ctx.get("addresses")
+        if addr_config.get("bano", {}).get("file"):
+            logging.info("importing bano addresses")
+            conf = ctx.addresses.bano
+            additional_params = _get_cli_param(conf.get("nb_threads"), "--nb-threads")
+            additional_params += _get_cli_param(conf.get("nb_shards"), "--nb-shards")
+            additional_params += _get_cli_param(conf.get("nb_replicas"), "--nb-replicas")
+            run_rust_binary(
+                ctx,
+                "mimir",
+                "bano2mimir",
+                files,
+                "--input {ctx.addresses.bano.file} \
+                --connection-string {ctx.es} \
+                {additional_params} \
+                --dataset {ctx.dataset}".format(
+                    ctx=ctx, additional_params=additional_params
+                ),
+            )
+        if addr_config.get("oa", {}).get("path"):
+            logging.info("importing oa addresses")
+            conf = ctx.addresses.oa
+            additional_params = _get_cli_param(conf.get("nb_threads"), "--nb-threads")
+            additional_params += _get_cli_param(conf.get("nb_shards"), "--nb-shards")
+            additional_params += _get_cli_param(conf.get("nb_replicas"), "--nb-replicas")
+            run_rust_binary(
+                ctx,
+                "mimir",
+                "openaddresses2mimir",
+                files,
+                "--input {ctx.addresses.oa.path} \
+                --connection-string {ctx.es} \
+                {additional_params} \
+                --dataset {ctx.dataset}".format(
+                    ctx=ctx, additional_params=additional_params
+                ),
+            )
+        return
+
+    output_csv = "output.csv.gz"
+
+    options = []
+    if ctx.addresses.get("bano", {}).get("file"):
+        options.append("--bano")
+        options.append(ctx.addresses.bano.file)
+    if ctx.addresses.get("oa", {}).get("file"):
+        options.append("--openaddresses")
+        options.append(ctx.addresses.oa.file)
+    if ctx.get("osm", {}).get("file"):
+        options.append("--osm")
+        options.append(ctx.osm.file)
+    if len(options) == 0:
+        return
+    options.append("--output-compressed-csv")
+    options.append("/data/{}".format(output_csv))
+
+    logging.info("Running addresses importer/deduplicator")
+
+    run_rust_binary(ctx, "addresses-importer", "", files, ' '.join(options))
+
+    logging.info("Import addresses into ES instance")
+
+    run_rust_binary(
+        ctx,
+        "mimir",
+        "openaddresses2mimir",
+        files,
+        "--input /data/{output_csv} \
+        --connection-string {ctx.es} \
+        --dataset {ctx.dataset}".format(
```

Comment:
Some additional params are missing here, to pass the number of threads, shards, and replicas.  
(See the existing usage of the openaddresses importer)